cmd/netobserv-ebpf-agent.go

```diff
@@ -54,9 +54,6 @@ func main() {
 	logrus.WithField("configuration", fmt.Sprintf("%#v", config)).Debugf("configuration loaded")
 
 	if config.EnablePCA {
-		if config.PCAFilters == "" {
-			logrus.Info("[PCA] NetObserv eBPF Agent instantiated without filters to identify packets. All packets will be captured. This might cause reduced performance.")
-		}
```

Comment:
Do we still need a warning here in case filters are too wide ?

Reply:
That filter doesn’t exist anymore our option is the doc this as warning

Reply:
yeah we need to be explicit in the doc about perf impact and implement a safe guard in case the CLI collector is getting too much data.